spidermon/contrib/actions/email/ses.py

```diff
@@ -54,7 +54,7 @@ def _get_recipients(self):
                 recipients.extend(recipient_subset)
         return recipients
 
-    def send_message(self, message):
+    def send_message(self, message, **kwargs):
```

Comment:
**question** Why are you changing this class that is not related to the issue you are implementing?

Reply:
I needed to add callback and setting some argument such as debug in MailSender while doing testing. So I've to change the base send_message and to make sure that this class doesn't break with that change. That's why I've changed this method.